ras_party/controllers/respondent_controller.py

```diff
@@ -69,15 +71,25 @@ def get_respondents_by_name_and_email(first_name, last_name, email, page, limit,
     }
 
 
+def get_respondent_by_party_uuid(party_uuid: UUID, session: session) -> Respondent:
```

Comment:
At the moment this is quite fluid, I might wrap it in a session at some stage, but decided not to at present.